solr/modules/ltr/example/train_and_upload_demo_model.py

```diff
@@ -2,8 +2,8 @@
 
 import sys
```

Comment:
I can't add a comment to the first line 

but `#!/usr/bin/env python` could be `#!/usr/bin/env python3`